### PR TITLE
hw1

### PR DESCRIPTION
**Pull request by gassannov**
Я использовал датасет с предсказанием сердечных заболеваний по показателям здоровья. В качестве модели использовал GradientBoostingClassifier. Все категориальные данные уже были обработаны, поэтому мне не пришлоьс этого делать, а в предобработке, я просто нормализовал численные данные. В качестве метрик использовал accuracy и f1 меру.
Самооценка:
1-есть
2-есть, но только ноутбук
3-есть
4-есть
5-есть
6-есть
7-есть
8-генерировал рандомно без корреляций
9-есть
10-есть
11-есть
12-нет
13-нет
на дополнительные балы ничего нет

### CODE REVIEW

File: homework1/configs/config2.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+output_metric_path: "models/metrics.json"
+val_size: 0.2
+training_params:
+  loss: 'deviance'
+  learning_rate: 0.05
+  n_estimators: 200
+feature_params:
+  deleted_features: ['fbs']
+  target_column: 'condition'
+  normalize_features: ['age', 'trestbps', 'chol', 'thalach', 'oldpeak']
```
Comment (bulaevvi) [inline homework1/configs/config2.yaml:12]: В условии указано: "закоммитьте как минимум 2 корректные конфигурации, с помощью которых можно обучить модель (разные модели, стратегии split, preprocessing) (3 балла)". У Вас 2 корректные конфигурации, но модель одна и та же, стратегии разбиения и препроцессинга одинаковые. Поэтому ставлю только один балл за этот пункт

**Review by bulaevvi — APPROVED** [review]
Итого получилось 24 балла. Можете делать merge
